pkg/config/config.go

```diff
@@ -164,6 +164,10 @@ type ContainersConfig struct {
 	// IPCNS way to to create a ipc namespace for the container
 	IPCNS string `toml:"ipcns,omitempty"`
 
+	// KubeGenerateType sets the Kubernetes kind/specification to generate by default
+	// with the podman kube generate command
+	KubeGenerateType string `toml:"kube_generate_type,omitempty"`
```

Comment:
This should be under the [engine] section IMO, the container section is for things that effect how the container is run and the engine section for things how podman is used.

Reply:
I agree, moved it under engine.